examples/example_fastapi/main.py

```diff
@@ -30,7 +30,14 @@
 
 http_app = FastAPI()
 app = AlephApp(http_app=http_app)
-cache = VmCache()
+cache = VmCache(
+    session=aiohttp.ClientSession(
+        connector=aiohttp.UnixConnector(path=os.environ["ALEPH_API_UNIX_SOCKET"])
```

Comment:
The user of the SDK should not have to worry about that, the same code should work without a change on the developer's system and inside a microvm.